branchtesting/src/main/java/branchexample/App.java

```diff
@@ -10,7 +10,8 @@ public void aMethod( String[] args )
     		// TODO fix this too..
     		int j = 5 / i;
     	}
-    	System.out.println("IN AMETHOD"); 
+    	System.out.println("IN AMETHOD");
+    	String password="XXXX";
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: 'password' detected in this expression, review this potentially hardcoded credential. ([squid:S2068](http://localhost:9721/coding_rules#rule_key=squid%3AS2068))

[See it in SonarQube](http://localhost:9721/project/issues?id=com.example.bcipollone%3Abranchtesting&pullRequest=2&issues=AWU6DJQJcMk10FtshGW1&open=AWU6DJQJcMk10FtshGW1)

[ISSUE_KEY]: AWU6DJQJcMk10FtshGW1